tests/neo.UnitTests/Consensus/UT_Consensus.cs

```diff
@@ -100,35 +98,234 @@ public void ConsensusService_Primary_Sends_PrepareRequest_After_OnStart()
                     MerkleRoot = header.MerkleRoot,
                     Timestamp = header.Timestamp,
                     Index = header.Index,
-                    NextConsensus = header.NextConsensus
+                    NextConsensus = header.NextConsensus,
+                    Transactions = new Transaction[0]
                 }
-            });
-
+            };
+            Console.WriteLine("\n==========================");
+            Console.WriteLine("Telling a new block to actor consensus...");
+            Console.WriteLine("will trigger OnPersistCompleted without OnStart flag!");
             // OnPersist will not launch timer, we need OnStart
+            actorConsensus.Tell(testPersistCompleted);
+            Console.WriteLine("\n==========================");
 
+            Console.WriteLine("\n==========================");
             Console.WriteLine("will start consensus!");
             actorConsensus.Tell(new ConsensusService.Start
             {
                 IgnoreRecoveryLogs = true
             });
 
-            Console.WriteLine("OnTimer should expire!");
-            Console.WriteLine("Waiting for subscriber message!");
-            // Timer should expire in one second (block_received_time at :01, initialized at :02)
-
-            var answer = subscriber.ExpectMsg<LocalNode.SendDirectly>();
-            Console.WriteLine($"MESSAGE 1: {answer}");
-            //var answer2 = subscriber.ExpectMsg<LocalNode.SendDirectly>(); // expects to fail!
-
+            Console.WriteLine("Waiting for subscriber recovery message...");
+            var askingForInitialRecovery = subscriber.ExpectMsg<LocalNode.SendDirectly>();
+            Console.WriteLine($"Recovery Message I: {askingForInitialRecovery}");
+            ConsensusPayload initialRecoveryPayload = (ConsensusPayload)askingForInitialRecovery.Inventory;
+            RecoveryRequest rrm = (RecoveryRequest)initialRecoveryPayload.ConsensusMessage;
+            rrm.Timestamp.Should().Be(328665601001);
+
+            Console.WriteLine("Waiting for backupChange View... ");
+            var backupOnAskingChangeView = subscriber.ExpectMsg<LocalNode.SendDirectly>();
+            var changeViewPayload = (ConsensusPayload)backupOnAskingChangeView.Inventory;
+            ChangeView cvm = (ChangeView)changeViewPayload.ConsensusMessage;
+            cvm.Timestamp.Should().Be(328665601001);
```

Comment:
Can we put those timestamps into a variable?

Reply:
I think it is not needed, it will make the code less readable.
This is ok for now.

In the future there could be more complex logics using timestamp, then, we improve it.

Reply:
Why do you think that using a timestamp is more readable than using a variable name? You can create different variables if you need different timestamps.

Reply:
I thought they were different on each line.
However, they are repeated 3 times.

I moved to a variable. Thanks for the suggestion.